From 2287fce6adb7e62918bf06792acb13f1eac3f1f9 Mon Sep 17 00:00:00 2001
From: Rex Lu <rex.lu@mediatek.com>
Date: Wed, 29 Nov 2023 13:56:52 +0800
Subject: [PATCH 074/116] mtk: wifi: mt76: mt7996: Remove wed_stop during L1
 SER

Align logan L1 SER flow. During L1 SER, didn't need to close wed interrupt.

CR-Id: WCNCR00259341
Signed-off-by: Rex Lu <rex.lu@mediatek.com>
---
 mt7996/mac.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/mt7996/mac.c b/mt7996/mac.c
index b4703804b..0805251e8 100644
--- a/mt7996/mac.c
+++ b/mt7996/mac.c
@@ -1950,12 +1950,6 @@ void mt7996_mac_reset_work(struct work_struct *work)
 	dev_info(dev->mt76.dev,"\n%s L1 SER recovery start.",
 		 wiphy_name(dev->mt76.hw->wiphy));
 
-	if (mtk_wed_device_active(&dev->mt76.mmio.wed_hif2))
-		mtk_wed_device_stop(&dev->mt76.mmio.wed_hif2);
-
-	if (mtk_wed_device_active(&dev->mt76.mmio.wed))
-		mtk_wed_device_stop(&dev->mt76.mmio.wed);
-
 	ieee80211_stop_queues(mt76_hw(dev));
 	if (phy2)
 		ieee80211_stop_queues(phy2->mt76->hw);
-- 
2.39.2

